packages/docusaurus-theme-classic/codeTranslations/hi.json

```diff
@@ -0,0 +1,55 @@
+  "theme.NotFound.title": "पेज नहीं मिला",
+  "theme.Playground.liveEditor": "लाइव एडिटर",
+  "theme.Playground.result": "परिणाम",
+  "theme.PwaReloadPopup.closeButtonAriaLabel": "बंद करे",
+  "theme.PwaReloadPopup.info": "नया वर्जन उपलब्ध है",
+  "theme.PwaReloadPopup.refreshButtonText": "रिफ्रेश",
+  "theme.SearchBar.label": "खोज करें",
+  "theme.SearchPage.algoliaLabel": "अल्गोलिया द्वारा खोजें",
+  "theme.SearchPage.documentsFound.plurals": "एक डॉक्यूमेंट मिला|{count} डॉक्यूमेंट मिलें",
+  "theme.SearchPage.emptyResultsTitle": "डॉक्यूमेंटेशन में खोजें",
+  "theme.SearchPage.existingResultsTitle": "\"{query}\" के लिए खोज परिणाम",
+  "theme.SearchPage.fetchingNewResults": "नए परिणाम प्राप्त कियें जा रहे हैं...",
+  "theme.SearchPage.inputLabel": "खोज करें",
+  "theme.SearchPage.inputPlaceholder": "अपनी खोज यहाँ टाइप करें",
+  "theme.SearchPage.noResultsText": "कोई परिणाम नहीं मिलें",
+  "theme.blog.paginator.navAriaLabel": "ब्लॉग सूची पेज नेविगेशन",
+  "theme.blog.paginator.newerEntries": "नए एंट्रीज़",
+  "theme.blog.paginator.olderEntries": "पुराने एंट्रीज़",
+  "theme.blog.post.paginator.navAriaLabel": "ब्लॉग पोस्ट पेज नेविगेशन",
+  "theme.blog.post.paginator.newerPost": "नए पोस्ट",
+  "theme.blog.post.paginator.olderPost": "पुराने पोस्ट",
+  "theme.blog.post.plurals": "एक पोस्ट|{count} पोस्ट",
+  "theme.blog.post.readMore": "और पढ़ें",
+  "theme.blog.post.readingTime.plurals": "एक मिनट में पढ़ें|{readingTime} मिनट में पढ़ें",
+  "theme.blog.tagTitle": "{nPosts} पोस्ट \"{tagName}\" टैग के साथ",
+  "theme.common.editThisPage": "इस पेज को बदलें",
+  "theme.common.headingLinkTitle": "शीर्षक का सीधा लिंक",
+  "theme.common.skipToMainContent": "मुख्य कंटेंट तक स्किप करें",
+  "theme.docs.paginator.navAriaLabel": "डॉक्स पेज नेविगेशन",
+  "theme.docs.paginator.next": "अगला",
+  "theme.docs.paginator.previous": "पिछ्ला",
+  "theme.docs.sidebar.collapseButtonAriaLabel": "साइडबार बंद करें",
+  "theme.docs.sidebar.collapseButtonTitle": "साइडबार बंद करें",
+  "theme.docs.sidebar.expandButtonAriaLabel": "साइडबार खोलें",
+  "theme.docs.sidebar.expandButtonTitle": "साइडबार खोलें",
+  "theme.docs.sidebar.responsiveCloseButtonLabel": "मेन्यू बंद करें",
+  "theme.docs.sidebar.responsiveOpenButtonLabel": "मेन्यू खोलें",
+  "theme.docs.versions.latestVersionLinkLabel": "सबसे नया वर्जन",
+  "theme.docs.versions.latestVersionSuggestionLabel": "अप-टू-डेट डॉक्यूमेंटेशन के लिए {latestVersionLink} ({versionLabel}) देखें।",
+  "theme.docs.versions.unmaintainedVersionLabel": "यह {siteTitle} {versionLabel} के लिए डॉक्यूमेंटेशन है, जिसे अब सक्रिय रूप से नहीं बनाए रखा गया है।",
+  "theme.docs.versions.unreleasedVersionLabel": "यह {siteTitle} {versionLabel} वर्जन के लिए प्रकाशित डॉक्यूमेंटेशन है।",
+  "theme.lastUpdated.atDate": " {date} पर",
+  "theme.lastUpdated.byUser": " {user} द्वारा",
+  "theme.lastUpdated.lastUpdatedAtBy": "आखरी अपडेट {atDate} पर {byUser} द्वारा",
```

Comment:
Important note: this message will be injected with these 2 "inner messages":

![image](https://user-images.githubusercontent.com/749374/114310807-702eb280-9aec-11eb-9710-738806b6b6d1.png)

Note the inner messages are both optional, there can be a date, a user, both, or none (4 possibilities). If both date and user are present, your text would lead to duplicate whitespace as the inner messages already start with space.

Can you try to make it work so that there is no duplicate space and the text would work for the 4 possible cases?

Reply:
I'm not sure I follow, the spaces for these two were included in the `base.json` file so I did not remove them. Can you give an example to help me understand?

Reply:
I mean, the issue is prpobably in "lastUpdatedBy", not the 2 inner messages.

The original message has no spaces (on purpose): `  "theme.lastUpdated.lastUpdatedAtBy": "Last updated{atDate}{byUser}",`

Reply:
oh got it, sorry I completely missed it! I will make the change